react/components/MinicartIconButton.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { ButtonWithIcon } from 'vtex.styleguide'
+import { IconCart } from 'vtex.store-icons'
+import { useOrderForm } from 'vtex.order-manager/OrderForm'
+import { useCssHandles } from 'vtex.css-handles'
+import { useMinicartDispatch, useMinicartState } from '../MinicartContext'
+
+const CSS_HANDLES = ['minicartIconContainer', 'minicartQuantityBadge'] as const
+
+const MinicartIconButton = () => {
+  const {
+    orderForm,
+    loading,
+  }: { loading: boolean; orderForm: OrderForm } = useOrderForm()
```

Comment:
Could use OrderFormContext just like it's being done in the other component, right?

Reply:
Yep! It should!